### PR TITLE
Annotations

### PR DESCRIPTION
**Pull request by neithernut**
The branch reflects changes made due to annotations we got from our supervisor.

### CODE REVIEW

File: paper/src/transformation.tex [hunk 1]
```diff
@@ -72,7 +74,7 @@ \subsection{Static evaluation}
 C++ \cite{veldhuizen2003c++}.
 Thus, one can process information embedded in types at compile time and use the
 result at run time or to abort the compilation if a condition is not
-met\footnote{e.g. using \mintinline{cpp}{static_assert} in modern C++.}.
+met, e.g. using \mintinline{cpp}{static_assert} in modern C++..
```
Comment (matthiasbeyer) [inline paper/src/transformation.tex]: Doubled `.` at the end of the line(?)

Comment (neithernut) [reply paper/src/transformation.tex]: Yep, will fix that soonish.

**Comment by neithernut** [comment]
The one annotation I din not fixed is the first one showing up in the abstract.  The sentence is copies from the introduction. So it has to be fixed in both places. Could you have a look, @matthiasbeyer?

And we still have to get @Twonky to read his mails.

**Comment by matthiasbeyer** [comment]
~~I'm not entirely sure what you mean, sorry.~~

Ah, I see it now. From what I read, the sentence is not wrong.

> The Object Constraint Language (OCL) is used to annotate the code of a program with constraints that must be resolved by a resolution engine.

but, as I understand it, not as precise as wanted. Maybe something like


> The OCL is used to annotate the code of a program with constraints to model requirements of a program. Those annotations are resolved by a resolution engine.

Not entirely sure, though.

**Comment by neithernut** [comment]
> requirements

This suggests something completely different. However, if we swap this part, we should be fine. Maybe I'll push an alternative tomorrow.

**Comment by matthiasbeyer** [comment]
I don't think @Twonky will ever notice this, so I guess we leave him out.

**Comment by Twonky** [comment]
Oh I did notice it, just don't have anything constructive to add

**Comment by matthiasbeyer** [comment]
So you're fine with publishing the paper?

**Comment by Twonky** [comment]
yeah sure

**Comment by neithernut** [comment]
Finally, found some time to finish this. Please review.
Note: formatting, if any, should be done in a separate PR based on this one.

**Comment by neithernut** [comment]
Btw: if we are finished (including formatting), I will write an email to our supervisor.

**Pull request merged**